routing/result_interpretation.go

```diff
@@ -626,21 +662,21 @@ func (i *interpretedResult) successPairRange(
 
 // getPair returns a node pair from the route and the amount passed between that
 // pair.
-func getPair(rt *route.Route, channelIdx int) (DirectedNodePair,
+func getPair(rt *mcRoute, channelIdx int) (DirectedNodePair,
```

Comment:
Nice that this is already abstracted away behind the `DirectedNodePair` struct, so the diff can be pretty minimal.